pkgs/development/libraries/physics/geant4/datasets.nix

```diff
@@ -1,22 +1,32 @@
 { stdenv, fetchurl, }:
 
 let
-  mkDataset = { name, version, sha256, envvar}:
+  mkDataset = { name, version, sha256, envvar, priority ? 5}:
     stdenv.mkDerivation {
       inherit name version;
 
       src = fetchurl {
-        url = "https://geant4-data.web.cern.ch/geant4-data/datasets/${name}.${version}.tar.gz";
+        url = "https://cern.ch/geant4-data/datasets/${name}.${version}.tar.gz";
         inherit sha256;
       };
 
       preferLocalBuild = true;
       dontBuild = true;
       dontConfigure = true;
 
+      # Fix conflict between READMEs of various packages
       installPhase = ''
         mkdir -p $out/data
         mv ./* $out/data
+        fixConflicts() {
+          file=$1
+          if [[ -e $out/data/$file ]];
+            then
+            mv $out/data/$file $out/data/$file.${name}.${version};
+          fi
+        }
+        fixConflicts README
+        fixConflicts History
```

Comment:
A more robust approach would be to instead change installation path from "$out/data" to something of the form like "$out/share/Geant4-10.5.1/data/G4ABLA3.1" (I took this path from the "bin/geant4.sh").

Reply:
(Such change would probably deserve it's own PR)

Reply:
that's a great idea!